faux-mgs/src/main.rs

```diff
@@ -1720,6 +1741,51 @@ async fn run_command(
                 }
             }
         },
+        Command::Ereports { start_ena, committed_ena, restart_id, limit } => {
+            let tranche = sp
```

Comment:
Tiny nit - maybe

```suggestion
            let EreportTranche { restart_id, ereports } = sp
```

so any new fields get included in this output?